integration-cli/docker_cli_search_test.go

```diff
@@ -16,10 +16,20 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 }
 
 func (s *DockerSuite) TestSearchStarsOptionWithWrongParameter(c *check.C) {
-	out, _, err := dockerCmdWithError("search", "--stars=a", "busybox")
+	out, _, err := dockerCmdWithError("search", "--filter has-stars=a", "busybox")
```

Comment:
you have to separate the flag name and the value in two different strings, like the example below. Otherwise, it thinks `--filter has-stars=a` is the full name of the flag.

```
dockerCmdWithError("search", "--filter", "has-stars=a", "busybox")
```